_shared/project/README.md

```diff
@@ -0,0 +1,263 @@
+{% if cookiecutter.get("visibility") == "public" -%}
+<a href="{{ cookiecutter.__github_url }}/actions/workflows/ci.yml?query=branch%3Amain"><img src="https://img.shields.io/github/workflow/status/{{ cookiecutter.github_owner }}/{{ cookiecutter.slug }}/CI/main"></a>
+{% endif -%}
+{% if cookiecutter._directory == 'pypackage' -%}
+<a href="{{ cookiecutter.__pypi_url }}"><img src="https://img.shields.io/pypi/v/{{ cookiecutter.slug }}"></a>
+{% endif -%}
+<a><img src="https://img.shields.io/badge/python-{{ python_versions()|pyformat(PyFormats.MAJOR_DOT_MINOR_FMT)|separator(" | ") }}-success"></a>
+<a href="{{ cookiecutter.__github_url }}/blob/main/LICENSE"><img src="https://img.shields.io/badge/license-BSD--2--Clause-success"></a>
+<a href="https://github.com/hypothesis/cookiecutters/tree/main/{{ cookiecutter._directory }}"><img src="https://img.shields.io/badge/cookiecutter-{{ cookiecutter._directory|replace("-", "--") }}-success"></a>
+<a href="https://black.readthedocs.io/en/stable/"><img src="https://img.shields.io/badge/code%20style-black-000000"></a>
+
+# {{ cookiecutter.name }}
+
+{{ cookiecutter.short_description }}
+
+{{- include("README/head.md") }}
+
+{%- if cookiecutter.get("console_script") == "yes" %}
+
+## Installing
+
+We recommend using [pipx](https://pypa.github.io/pipx/) to install
+{{ cookiecutter.name }}.
+First [install pipx](https://pypa.github.io/pipx/#install-pipx) then run:
+
+```terminal
+pipx install {{ cookiecutter.slug }}
+```
+
+You now have {{ cookiecutter.name }} installed! For some help run:
+
+```
+{{ cookiecutter.__entry_point }} --help
+```
+
+## Upgrading
+
+To upgrade to the latest version run:
+
+```terminal
+pipx upgrade {{ cookiecutter.slug }}
+```
+
+To see what version you have run:
+
+```terminal
+{{ cookiecutter.__entry_point }} --version
+```
+
+## Uninstalling
+
+To uninstall run:
+
+```
+pipx uninstall {{ cookiecutter.slug }}
+```
+{%- endif %}
```

Comment:
Without indentation it's not easy to see what this is an if around? There's a suggestion below about splitting things out into separate files and including the whole lot. You could do something similar here.